.stylelintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "stylelint-config-recommended",
+  "rules": {
+    "at-rule-no-unknown": [
```

Comment:
Could https://www.npmjs.com/package/stylelint-scss negate the need for this rule?